spec/integration/user_index_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe 'User Index Page', type: :system do
```

Comment:
- [ ] specs file for the integration tests type is `feature`, for example
```ruby
describe 'the signin process', type: :feature do
  ...
 end
```

Reply:
I have made the changes.